packages/frontend/src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -227,7 +227,7 @@ class SetupRecoveryMethod extends Component {
             const { secretKey } = parseSeedPhrase(recoverySeedPhrase);
             const recoveryKeyPair = KeyPair.fromString(secretKey);
             await validateSecurityCode(accountId, method, securityCode, validateSecurityCodeEnterpriseRecaptchaToken, 'setupRecoveryMethodNewAccount');
-            await saveAccount(accountId, recoveryKeyPair);
+            await wallet.saveAccountKeyPair({ implicitAccountId: accountId, recoveryKeyPair });
```

Comment:
Seems like it should be:
`await wallet.saveAccountKeyPair({ accountId, recoveryKeyPair });`

Reply:
Great catch! Fixed